mariadb/rootfs/etc/confd/templates/mariadb-server.tmpl

```diff
@@ -8,7 +8,8 @@
 # this is only for the mysqld standalone daemon
 [mysqld]
 # skip-networking
-
+# Allow for max_allowed_packet to be set in the project .env
+max_allowed_packet ={{ getenv "DB_MAX_ALLOWED_PACKET" }}
```

Comment:
Remove the comment as it is not accurate.

Change the line to:

```
max_allowed_packet={{ getenv "MYSQL_MAX_ALLOWED_PACKET" }}
```

Also don't forget to rename the file as is stated in an earlier comment.